src/modules/app/browser-app.module.ts

```diff
@@ -51,12 +50,12 @@ export function getRequest(transferState: TransferState): any {
     HttpClientModule,
     // forRoot ensures the providers are only created once
     IdlePreloadModule.forRoot(),
-    RouterModule.forRoot([], {
-      // enableTracing: true,
-      useHash: false,
-      scrollPositionRestoration: 'enabled',
-      preloadingStrategy: NoPreloading
-    }),
+    // RouterModule.forRoot([], {
```

Comment:
why is this section commented?

Reply:
Also to test something. I've reverted it. Thanks for catching that!